CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ if (TREMOTESF_BUILD_TESTS)
     enable_testing()
 endif()
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
That's needed for some IDEs, right? I think it would be better to split this in separate PR (or at least separate commit).

Reply:
Yup I use it with vim-lsp for code navigation. I'll remove it from this PR but if you're open to adding it then it makes my life easier :-)